protocolAggregator/enclave/src/main/java/com/protocol180/aggregator/enclave/AggregationEnclave.java

```diff
@@ -134,16 +130,12 @@ protected final void receiveMail(EnclaveMail mail, String routingHint) {
 
 
         MailType mailType = getMailType(new String(unencryptedMail));
-        System.out.println("Type of mail for current request is:" + mailType);
-
-        String senderEncodedPublicKey = Base64.getEncoder().encodeToString(mail.getAuthenticatedSender().getEncoded());
-        System.out.println("Clients randomly generated public key is: " + senderEncodedPublicKey);
 
         try {
             if (mailType.equals(MailType.TYPE_PROVIDER)) {
                 clientTypeForCurrRequest = MailType.TYPE_PROVIDER.type;
                 //store mail contents for aggregation
-                System.out.println("Ack Mail for provider");
+                System.out.println("Provider mail received.");
```

Comment:
lets remove system prints to log.info or log.debug

Reply:
log.info don't work in enclave. Enclave can only pass message to console through sys.out or sys.err